atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -59,8 +64,20 @@ public boolean enableOracleEnterpriseFeatures() {
         return () -> new OracleDbTableFactory(this);
     }
 
+    @Value.Default
+    @Override
+    public String tablePrefix() {
+        return "a_";
+    }
+
     @Override
     public final String type() {
         return TYPE;
     }
+
+    @Value.Check
+    protected final void check() {
+        Preconditions.checkState(tablePrefix() != null && !tablePrefix().isEmpty(),
+                "'tablePrefix' must not be an empty string");
```

Comment:
Yes.. I am enforcing that the ``tablePrefix`` doesnt begin with underscore. This is definitely a better check, also we should check that ``tableNames`` and all prefixes have alphanumeric characters and underscores only.